### ISSUE TITLE
Create intuitive graph for gradient descent in a multidimensional parameter space

### ISSUE THREAD

**Issue by chiroptical** [chiroptical/pytorch-introduction#3]
Build a conceptual image for gradient descent
